src/nvim/api/vim.c

```diff
@@ -127,6 +127,9 @@ void nvim_feedkeys(String keys, String mode, Boolean escape_csi)
 /// Unlike `nvim_feedkeys`, this uses a lower-level input buffer and the call
 /// is not deferred. This is the most reliable way to emulate real user input.
```

Comment:
_implement_ user input, not "emulate". Or just "send user input". The API _does_ things, it doesn't emulate doing things.

Reply:
I'll change it, though I sympathize with the original wording: I think it was to contrast e.g. feedkeys() which is not equivalent to raw user input.

Reply:
I didn't argue against that part. Removing "emulate" makes that point stronger, it indicates something slightly removed from the real thing, when it is in fact the real thing.